src/app/static/src/tests/components/runWorkflow/runWorkflowProgress.test.ts

```diff
@@ -162,7 +163,7 @@ describe(`runWorkflowProgress`, () => {
             expect(runningStatus.classes()).toContain("text-secondary")
 
             const dateColumns = wrapper.findAll("tr > td:nth-child(3)")
-            expect(dateColumns.at(0).text()).toBe("Wed Jun 16 2021, 09:51")
+            expect(dateColumns.at(0).text()).toBe(longTimestamp(new Date("2021-06-16T09:51:16Z")))
```

Comment:
Has this behaviour actually changed? Feels like it was a slightly better test before when it was checking the actual expected text rather than calling the same helper (?) as the component.